diff -uNr node-v16.6.1/deps/v8/src/heap/base/asm/x64/push_registers_asm.cc node-v16.6.1.mod/deps/v8/src/heap/base/asm/x64/push_registers_asm.cc
--- node-v16.6.1/deps/v8/src/heap/base/asm/x64/push_registers_asm.cc	2021-08-03 05:54:24.000000000 +0000
+++ node-v16.6.1.mod/deps/v8/src/heap/base/asm/x64/push_registers_asm.cc	2021-08-08 03:01:21.030263715 +0000
@@ -16,6 +16,7 @@
 // GN toolchain (e.g. ChromeOS) and not provide them.
 // _WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64.
 // Otherwise, undefined.
+#ifndef __ANDROID__
 #ifdef _WIN64
 
 // We maintain 16-byte alignment at calls. There is an 8-byte return address
@@ -104,3 +105,4 @@
     "  ret                                              \n");
 
 #endif  // !_WIN64
+#endif
