diff -uNr node-v16.6.1/deps/v8/BUILD.gn node-v16.6.1.mod/deps/v8/BUILD.gn
--- node-v16.6.1/deps/v8/BUILD.gn	2021-08-03 05:54:24.000000000 +0000
+++ node-v16.6.1.mod/deps/v8/BUILD.gn	2021-08-07 15:26:13.212360709 +0000
@@ -10,6 +10,9 @@
 import("//build/config/sanitizers/sanitizers.gni")
 import("//build_overrides/build.gni")
 
+v8_current_cpu = getenv("DEST_CPU")
+current_cpu = getenv("DEST_CPU")
+
 if (is_android) {
   import("//build/config/android/rules.gni")
 }
